hack/tools.sh

```diff
@@ -138,7 +140,26 @@ install_crdoc() {
 	}
 	ok "crdoc was installed successfully"
 }
+install_oc() {
+	local version=" $OC_VERSION"
+	local os="$GOOS"
```

Comment:
this assume "$GOOS" is defined.  rather 

```suggestion
	local os
	os="$(go env GOOS)"
```

Reply:
Yes, but we already define GOOS and GOARCH above.